source/common/upstream/health_discovery_service.cc

```diff
@@ -54,12 +94,69 @@ void HdsDelegate::onReceiveInitialMetadata(Http::HeaderMapPtr&& metadata) {
   UNREFERENCED_PARAMETER(metadata);
 }
 
+void HdsDelegate::processMessage(
+    std::unique_ptr<envoy::service::discovery::v2::HealthCheckSpecifier>&& message) {
+  ENVOY_LOG(debug, "New health check response message {} ", message->DebugString());
+  ASSERT(message);
+
+  for (auto& cluster_health_check : message->health_check()) {
+    // Create HdsCluster config
+    envoy::api::v2::core::BindConfig bind_config;
+
+    clusters_config_.emplace_back();
```

Comment:
Why is this a field of the object? The scope looks limited to `processMessage()`.